core-plugins/src/main/java/co/cask/hydrator/plugin/batch/aggregator/RowDenormalizerConfig.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.hydrator.plugin.batch.aggregator;
+
+import co.cask.cdap.api.annotation.Description;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Splitter;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Config for RowDenormalizer Aggregator Plugin.
+ */
+public class RowDenormalizerConfig extends AggregatorConfig {
+
+  @Description("Name of the column in the input record which will be used to group the raw data. For Example: " +
+    "KeyField")
+  private final String keyField;
+
+  @Description("Name of the column in the input record which contains the names of output schema columns." +
+    "For example:" +
+    "Input Record has columns 'KeyField', 'FieldName', 'FieldValue'" +
```

Comment:
Only capitalize the first word in a sentence or proper nouns. Change to `input records have columns ...`